### ISSUE TITLE
Option to add small invisible fizzler´s frames on top and bottom part of the Fizzler Item to prevent "Exploits

### ISSUE THREAD

**Issue by LautaroL20** [BEEmod/BEE2-items#1652]
This would be pretty much useful in CO-OP Maps where you can exploit passing cubes under / over fizzlers.

![image](https://cloud.githubusercontent.com/assets/12244376/23574906/5bd48a1c-0063-11e7-8710-6952e46e7039.png)

**Comment by ZombiePwner1** [comment]
This is the technique i use in my hammer coop maps to prevent that annoying fizzler exploit. Also can i chat with you on steam @LautaroL20?

**Comment by LautaroL20** [comment]
@ZombiePwner1 - I have to use a custom fizzler instance (the one from the screenshot) in Beemod to prevent people from exploiting.

**Comment by ZombiePwner1** [comment]
@LautaroL20 lol and how come you went offline when i was trying to chat with you?

**Comment by ZombiePwner1** [comment]
@LautaroL20 Also i don't recommend using an instance since you can't stretch the little parts of the fizzler unless you edit the instance.

**Comment by LautaroL20** [comment]
Ive already edited the instance so it works correct.

**Comment by ZombiePwner1** [comment]
@LautaroL20 lol alright then anyways i'm waiting for you i wanna do some coop with you.

**Comment by ZombiePwner1** [comment]
@LautaroL20 Do the little fizzler trick for the checkpoints to prevent checkpoint exploits.

**Comment by LautaroL20** [comment]
Yes, ive already suggested the same for checkpoints, here: https://github.com/BEEmod/BEE2-items/issues/521

**Comment by Luke18033** [comment]
This would be a compiler issue, we'd want to have it automatically generate these around fizzlers that are touching walls.
